app/controllers/static_pages_controller.rb

```diff
@@ -0,0 +1,12 @@
+class StaticPagesController < ApplicationController
+  def home
+  end
```

Comment:
action ko có phần thân thì khai báo trên 1 line như này thôi em
```suggestion
  def home; end
```